### PR TITLE
Add implementation of standard deviation on data

### PR DESCRIPTION
**Pull request by th8917**
added standard deviation features

### CODE REVIEW

**Review by adam-parker1 — COMMENTED** [review]
LOOKS GREAT!

File: inflammation-analysis.py [hunk 1]
```diff
@@ -27,7 +27,8 @@ def main(args):
     for filename in InFiles:
         inflammation_data = models.load_csv(filename)
 
-        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data)}
+        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data), **(models.s_dev(inflammation_data))}
```
Comment (th8917) [inline inflammation-analysis.py:31]: The new entry in the dictionary **(models.s_dev(inflammation_data)) needs to be documented

File: inflammation/models.py [hunk 2]
```diff
@@ -54,3 +54,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```
Comment (adam-parker1) [inline inflammation/models.py:64]: Should have used np.std instead of your own function

File: inflammation-analysis.py (hunk 1 above)
Comment (adam-parker1) [reply inflammation-analysis.py:31]: agreed!

File: inflammation/models.py [hunk 3]
```diff
@@ -54,3 +54,13 @@ def daily_min(data):
...
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
```
Comment (adam-parker1) [inline inflammation/models.py]: s_dev2 is not a good name for a variable

Comment (th8917) [reply inflammation/models.py]: okay I will change it.

**Pull request merged**